lucille-core/src/main/java/com/kmwllc/lucille/stage/ParseDate.java

```diff
@@ -40,20 +37,48 @@ public class ParseDate extends Stage {
   private final List<String> sourceFields;
   private final List<String> destFields;
   private final UpdateMode updateMode;
+  private final String timeZoneId;
+
+  private static final String DEFAULT_TIME_ZONE_ID = Calendar.getInstance().getTimeZone().getID();
+//  private static final String DEFAULT_TIME_ZONE_ID = "UTC";
 
   public ParseDate(Config config) {
     super(config, new StageSpec().withRequiredProperties("source", "dest")
-        .withOptionalProperties("format_strs", "update_mode", "formatters"));
+        .withOptionalProperties("format_strs", "update_mode", "formatters", "time_zone_id"));
 
     this.formatters = new ArrayList<>();
-    this.formats = ConfigUtils.getOrDefault(config, "format_strs", new ArrayList<String>())
-        .stream()
-        .map(formatString -> new SimpleDateFormat(formatString))
-        .collect(Collectors.toUnmodifiableList());
 
     this.sourceFields = config.getStringList("source");
     this.destFields = config.getStringList("dest");
     this.updateMode = UpdateMode.fromConfig(config);
+
+    timeZoneId = getTimeZoneId(config);
+    formats = getFormats(config);
+  }
+
+  private static String getTimeZoneId(Config config) {
+    String timeZoneId = ConfigUtils.getOrDefault(config, "time_zone_id", DEFAULT_TIME_ZONE_ID);
+    if (Arrays.stream(TimeZone.getAvailableIDs()).noneMatch(timeZoneId::equals)) {
+      throw new IllegalArgumentException("Invalid time zone ID: \"" + timeZoneId
+          + "\", must be one of TimeZone.getAvailableIDs()");
+    }
+    return timeZoneId;
+  }
+
+  private List<DateFormat> getFormats(Config config) {
+
+    // convert format strings to a set to remove duplicates
+    Set<String> formatStrings = new HashSet<>(ConfigUtils.getOrDefault(config, "format_strs", new ArrayList<>()));
```

Comment:
We should preserve the order of the formatStrings. You could do this by using a LinkedHashSet instead of a HashSet here, but I would recommend `strList.stream().distinct().collect(Collectors.toList())` instead